neurom/view/view.py

```diff
@@ -156,14 +157,18 @@ def plot_soma(ax, soma, plane='xy',
                                    ignore=False)
 
 
-def plot_neuron(ax, nrn, plane='xy',
+# pylint: disable=too-many-arguments
+def plot_neuron(ax, nrn,
+                neurite_type=NeuriteType.all,
```

Comment:
this really changes the signature of the method: anyone not using kwargs is going to be hit by this.

Reply:
Actually, the plot_ methods should not be public as neurom.viewer.draw is supposed to be the interface to all of them.

Reply:
@mgeplf Given the time it will take before NeuroM reaches a new major version I think we should still merge this nonetheless.
Reasons:
1) `plot_*` methods are not public
2) Even if it breaks someone's code (which I am not even sure), it is a trivial fix.
3) The issue it solves has been mentionned once again: https://github.com/BlueBrain/NeuroM/issues/670#issuecomment-442819943

Reply:
ok, fine with me